pontoon/base/tests/test_models.py

```diff
@@ -80,6 +84,102 @@ def test_repository_for_path(self):
         path = os.path.join(repo2.checkout_path, 'foo', 'bar')
         assert_equal(project.repository_for_path(path), repo2)
 
+    def locales_parts_stats_helper(self):
```

Comment:
This is a nice reduction in repeated code; perhaps putting it in a `setUp` method would be even better? Rather than making tests call it.

If you decide to do that I'd suggest making a separate test class for it so the `setUp` doesn't interfere with tests that don't need it. Although I never really thought about it until now, I don't see why having multiple test classes for a single class to be tested is a bad thing if you use it for stuff like that.